### PR TITLE
Update Main Page (README.md) Include Mission Statement and Vision

### PR DESCRIPTION
**Pull request by swzCuroverse**
Updated README.md to include newly creatad mission statement and vision for the CWL project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,21 @@ collaborative open standards development](https://open-stand.org/about-us/princi
 the elected [CWL leadership team](#Leadership_Team), however every-day project decisions are made by the
 [CWL community](#Support) which is open for participation by anyone.
 
+## Vision and Mission Statement
+
+### CWL Vision
```
Comment (mr-c) [inline README.md:58]:
```suggestion
## Vision for the CWL Project
_What this community hopes to have accomplished in the future_
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,21 @@ collaborative open standards development](https://open-stand.org/about-us/princi
...
+
+Researchers, scientists, and analysts share their batch data analysis workflows without technical barriers using an open standard.  Sharing workflows this way is a usual occurrence and seen as a typical way of working.  The workflows are complete and run on a variety of environments; and people re-use shared workflow descriptions and build new workflows from them. No vendor dominates the ecosystem
```
Comment (mr-c) [inline README.md:60]:
```suggestion
Researchers, scientists, and analysts share their batch data analysis workflows without
technical barriers using an open standard.  Sharing workflows this way is a usual occurrence
and seen as a typical way of working.  The workflows are complete and run on a variety of
environments; and people re-use shared workflow descriptions and build new workflows from
them. No vendor dominates the ecosystem.
```

Line wrapping, added a trailing period to the last sentence.

File: README.md [hunk 3]
```diff
@@ -53,6 +53,21 @@ collaborative open standards development](https://open-stand.org/about-us/princi
...
+
+### CWL Mission Statement 
+
+To support open consensus-based standards for command line data analysis workflows and tools.
+
+Specifically, we support the
+* pre-standards process by providing a neutral place of convening to discuss, propose and test ideas about workflow standards
+* standardization process by stewarding the development and delivery of standards in accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
+* post-standards life cycle by promoting the released standard, developing and maintaining related training and tools, and by tracking deficits and other post-standardization feedback.
```
Comment (mr-c) [inline README.md:69]:
```suggestion
### Mission of the CWL Project
_How we plan to achieve our vision_

The CWL project supports open consensus-based standards for command line data analysis
workflows and tools.

Specifically, we support the
* pre-standards process by providing a neutral place of convening to discuss, propose and
test ideas about command-line tool based workflow standards and related topics
* standardization process by stewarding the development and delivery of standards in
accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
* post-standards life cycle by (1) promoting the released standards, (2) developing and maintaining
related training and tools, and by (3) tracking deficits and other post-standardization feedback.
```

Line wrapping, added "command-line tool based" before "workflows" to narrow the scope. Added "and related topics" to expand the scope a bit as well :-)

I think I will also move the OpenStand logo down to this section
"release standards" to "released standard**s**"

**Review by mr-c — CHANGES_REQUESTED** [review]
Thanks for moving the drafts here and for facilitating that process!

I suggested some small tweaks.

File: README.md (hunk 1 above)
Comment (swzCuroverse) [reply README.md:58]: Looks good.

File: README.md (hunk 2 above)
Comment (swzCuroverse) [reply README.md:60]: Looks good.

File: README.md (hunk 3 above)
Comment (swzCuroverse) [reply README.md:69]: Looks good.

File: README.md (hunk 1 above)
Comment (swzCuroverse) [reply README.md]: I merged these in.

**Review by mr-c — APPROVED** [review]

**Review by geoffjentry — APPROVED** [review]

**Review by tetron — APPROVED** [review]

**Comment by mr-c** [comment]
Since the website has had some technical changes since November, this PR is just to collect the final consent for the exact wording from the CWL Leadership Team members. Once that is obtained we'll move the text to appropriate location and reference the approvals collected here.

The mission and vision statement was reviewed publicly (and feedback requested) at the [2020-10-13](https://docs.google.com/document/d/1zzwirsc2esUWZMxDB9mFnGf2err1viJ9wELy-wIvwaI/edit?usp=sharing), [2020-11-03](https://docs.google.com/document/d/19sjukd3RW5G-IJ1vpQmR4YYw0RWgk1UlkVFRXHEelcs/edit#heading=h.wdp7w8xg6v3n), [2020-11-24](https://docs.google.com/document/d/1zzwirsc2esUWZMxDB9mFnGf2err1viJ9wELy-wIvwaI/edit?usp=sharing), [2020-12-01](https://docs.google.com/document/d/134HgU-tbzoRGpVf2Ke38CBHVKRig246YpT4yMh19hkA/edit?usp=sharing), [2020-12-08](https://docs.google.com/document/d/1dTuQO1BCXlKDgIKQxyvQwxd0sEvByvl14BkOiCXOyzM/edit?usp=sharing), and [2020-12-15](https://docs.google.com/document/d/14SOMXwBuq66WoPBcFg-bengGq9vNDOWpAJLwZ7RG8Ho/edit?usp=sharing) CWL Project meetings

**Review by bdusenbery — APPROVED** [review]

**Comment by mr-c** [comment]
@lunacodes Can you adapt this PR so that it works with the new website?

**Review by jmchilton — APPROVED** [review]

**Pull request closed without merging**